templates/includes/card.html

```diff
@@ -15,17 +15,19 @@
 -->
 
 {% from './pill.html' import pills %} {% macro card(api, type) %}
-<li id="{{api.title | slug}}" class="card" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}">
+<li id="{{api.title | slug}}" class="card" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}" data-iwa="{{api.iwa}}">
   <div class="card--header">
     <button class="card--more card--title">
       <h2 class="card--title">
-        {% if api.pwa %}
+        {% if api.iwa %}
+        <img class="card--location" src="/images/iwa.svg" alt="IWA" title="IWA" />
+        {% elseif api.pwa %}
```

Comment:
Shall we let chromium team define PWA and IWA at the same time?

Reply:
I guess that's a bigger branding option. For now, it's IWA, analog to PWA.